x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -150,6 +159,23 @@ export class SpaceSelector extends Component<Props, State> {
                 </EuiText>
               </Fragment>
             )}
+
+            {!this.state.loading && this.state.error && (
+              <Fragment>
+                <EuiSpacer />
+                <EuiText
+                  color="subdued"
+                  // @ts-ignore
+                  textAlign="center"
+                >
+                  <FormattedMessage
+                    id="xpack.spaces.spaceSelector.errorLoadingSpacesDescription"
+                    defaultMessage="Error loading spaces"
+                  />
+                  &nbsp;({this.state.error.message})
```

Comment:
optional nit: you can insert the dynamic content via the `values` prop
```suggestion
                  <FormattedMessage
                    id="xpack.spaces.spaceSelector.errorLoadingSpacesDescription"
                    defaultMessage="Error loading spaces ({message})"
                    values={{ message: this.state.error.message }}
                  />
```

Reply:
Ah nice! I really didn't like that `&nbsp;` hack, but had the feeling someone would correct me 😅